Automattic/vip-cli
# Kickoff import to Parker

## Description

Once everything looks good regarding the pre-signed URLs (#585), kickoff the import by calling Parker.

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `./dist/bin/vip-cookies.js nom`
